### ISSUE TITLE
Run the transcribe-then-oracle experiment against the corrected corpus

### ISSUE THREAD

**Issue by caiotheodoro** [caiotheodoro/specula#1]
## The question

Can the model's own panel transcription, fed into `verify()`, decide compliance well enough to ship — making end-to-end SFT unnecessary?

The end-to-end target asks one forward pass to do perception **and** ~91 serial rule decisions (15 of them divisions with three-branch rounding) before the first output token, with `enable_thinking=False`. The overfit probe showed that composite is not learnable even on its own training rows: 200 rows × 8 epochs, evaluated on **those same rows**, reached severity-weighted recall 0.156 at 384px and 0.282 at 768px. It never emitted PASS on any of 45 clean labels, and ~40% of its headline recall was reproducible by shuffling predictions against inputs.

Splitting the task removes the composite:

```
panel image --[model]--> structured fields --[verify()]--> Verdict
```

The model does only what a model is good at (read glyphs). Every rule, lookup table and division runs in Python, deterministically.

## Why it's worth running first

It costs **~200 inference passes, under 30 GPU-minutes, and zero training**. If it passes, the product ships without training anything.

Supporting evidence already measured:
- Single-line transcription accuracy was **0.977** at native resolution (study-17).
- Conditional on naming the right violation type, everything downstream was already fine — severity 0.96, `observed` string similarity 0.93. **Detection is the bottleneck, not the output format.**
- The pipeline's ceiling is now measured, not assumed: `verify(rebuild_label(gold_transcript(label))) == expected` on **60/60** tasks. Any model score reads against that bar.

## Pre-registered criteria — fixed before the run

| outcome | criterion | decision |
|---|---|---|
| **PASS** | transcribe→oracle micro-F1 ≥ 0.85 | Ship the two-stage pipeline. Cancel end-to-end SFT. |
| **PARTIAL** | 0.50 ≤ F1 < 0.85 | Retarget SFT to panel transcription only; keep the oracle in Python. |
| **FAIL** | F1 < 0.50 **and** per-field accuracy < 0.90 | Perception floor is real; fix the renderer/resolution before training. |
| **FAIL (surprising)** | F1 < 0.50 **with** per-field ≥ 0.95 | The analysis is wrong; investigate before spending GPU. |

**Report per-field transcription accuracy separately.** It is oracle-independent and therefore the durable result.

## How to run

```
modal run --detach cloud/modal_transcribe_probe.py \
  --tasks <stratified 200-row task file> \
  --adapter /checkpoints/sft-7b-v4 --name tx-corrected --limit 200
```

Stratify the probe set across all 13 RACC categories. The earlier set was 191/200 beverages, which let a single memorised RACC row carry `SERVING_SIZE`.

## Known caveats

- **Run it against the corrected corpus.** A previous attempt measured old-style panels against the fabricated `rounded_dv` grid and died at 16/200; those numbers are void.
- **The adapter may fight the instruction.** `sft-7b-v4` was fine-tuned to always emit verdict JSON, so a poor result is ambiguous between "cannot read" and "cannot be steered off its trained output format." Re-run against the **base** Qwen2.5-VL before concluding anything about perception.
- **The model does not sweep the panel.** In the partial run it returned 5–11 of 15 nutrient rows while closing valid JSON — not truncation, omission. The oracle then fabricates `MISSING_NUTRIENT` against rows the model never mentioned, which is what held precision at 0.36 against recall 0.86. Forcing enumeration in the prompt is the first thing to try if this comes back PARTIAL.

## Context

`docs/DECISIONS.md` study-17 (perception probe), study-19 (overfit probe), study-20 (formulation). Pipeline code: `model/src/specula_model/transcribe.py`, `cloud/modal_transcribe_probe.py`.
